management/backup/locations_test.go

```diff
@@ -203,6 +204,166 @@ func TestListLocations(t *testing.T) {
 	assert.True(t, found, "Expected location not found")
 }
 
+func TestUpdateLocation(t *testing.T) {
+	t.Parallel()
+	client := backupClient.Default.Locations
+
+	checkChange := func(t *testing.T, req locations.ChangeLocationBody, locations []*locations.LocationsItems0) {
+		t.Helper()
+		found := false
```

Comment:
Typically variables with default value in our code are initialized as `var found bool`